process-controller/src/main/java/org/jboss/as/process/Main.java

```diff
@@ -221,7 +222,9 @@ public JBossThreadFactory run() {
         final InetSocketAddress boundAddress = processController.getServer().getBoundAddress();
 
         final List<String> initialCommand = new ArrayList<String>();
-        initialCommand.add(jvmName);
+        final JdkType jdkType = JdkType.createFromJavaExecutable(jvmName);
+        initialCommand.add(jdkType.getJavaExecutable());
+        initialCommand.addAll(jdkType.getDefaultArguments());
```

Comment:
Is this necessary? The PC and the HC use the same VM.  And generally they share JAVA_OPTS as well, with domain.conf just providing a hook for setting up a jdwp only on one or the other.

We shouldn't be overriding user provided options with defaults (e.g. we think --illegal-access=permit is great but they do not and explicitly say so.) But checking for that here is starting to get complicated and ProcessController needs to be as simple as possible.

Reply:
Of course it is not necessary @bstansberry - removing it.